src/Pages/ResultList/Component/ResultListContents.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const ResultListContents = () => {
+  return RESTAURANT_LIST.map(data => (
```

Comment:
```suggestion
  return RESTAURANT_LIST.map(({
        id,
        detailUrl,
        address,
        imageUrl,
        dayOff,
        title,
        branch,
        searchPoint,
        place,
        category,
        viewCount,
        reviewCount,
      })=> (
```
데이터에 프로퍼티가 많을 경우 위와 같이 구조분해 할당해서 사용할 수 있습니다.